docs/references/protocol/data-types/permission-values.md

```diff
@@ -0,0 +1,84 @@
+---
+seo:
+    description: Format for permissions that can be granted to other accounts.
+label:
+    - Permissions
+---
+# Permission Values
+
+[Permission delegation](/docs/concepts/accounts/permission-delegation.md) defines permissions that can be granted to other accounts. These permissions fall into the following categories:
+
+- **Transaction Type Permissions** - Permission to send transactions with the specified [transaction type](../transactions/types/index.md).
+- **Granular Permissions** - Permission to send transactions with a specific subset of functionality.
+
+_(Requires the [PermissionDelegation amendment][] {% not-enabled /%}.)_
+
+## Numeric and String Values
+
+In the [canonical binary format](../binary-format.md) for transactions and ledger data, permission values are stored in a numeric form (specifically, as a 32-bit unsigned integer). However, in JSON they can be specified and returned in string format for convenience, similar to how transaction type names (`TransactionType` fields) work.
+
+When specifying a permission value in JSON, you can use either the numeric value or the string value. When serving data, the server supplies the string value if it is known, and falls back to the numeric value otherwise.
+
+{% admonition type="warning" name="Caution" %}
+Not all client libraries support numeric PermissionValue types. In most cases, you should use the string names of the permissions you want to grant.
+{% /admonition %}
+
+- For **transaction type permissions**, the string is the name of the transaction type exactly (case-sensitive). For example, a permission value of `"PaymentChannelClaim"` grants permission to send [PaymentChannelClaim transactions][].
+- For **granular permissions**, the string is the name of the granular permission (case-sensitive). For example, a permission value of `"TrustlineAuthorize"` grants permission to send TrustSet transactions that authorize trust lines (but not ones that modify other settings such as the trust line limit or freeze status).
+
+The numeric value `0` is reserved for "full permissions", meaning permission to send transactions of all types, but it is not possible to delegate full permissions.
+
+## Transaction Type Permissions
+
+Transaction Type Permissions have numeric values from 1 to 65536 (that is, 2<sup>16</sup>), inclusive. They correspond with known transaction types, except you add 1 when specifying a transaction type as a permission value. For example, the string `"Payment"` corresponds to a `TransactionType` value of `0`, but a `PermissionValue` value of `1`. To grant permissions to make Payment transactions, you can specify either `"PermissionValue": "Payment"` or `"PermissionValue": 1`.
+
+For a mapping of transaction types known by a server and their corresponding numeric transaction type values, check the `TRANSACTION_TYPES` field in the [server_definitions method][].
+
+### List of Non-Delegatable Permissions
```

Comment:
Maybe move this subsection under granular permissions.

Reply:
This doesn't belong under granular permissions, because the non-delegatable permissions are _transaction type_ permissions.